lib/dpul_collections/indexing_pipeline.ex

```diff
@@ -238,6 +238,25 @@ defmodule DpulCollections.IndexingPipeline do
     |> FiggyRepo.all()
   end
 
+  def get_figgy_parents(id) do
```

Comment:
I do wonder how hard it would be to do a query for parents like we do in Figgy. Would you mind if I gave it a shot?

Reply:
https://github.com/pulibrary/dpul-collections/pull/391 PR of what that looks like.

Reply:
Merged.